advanced_samples/CSV/dotnet/CSVtoOCS_pkce/appsettings.json

```diff
@@ -0,0 +1,8 @@
+﻿{
+  "NamespaceId": "",
```

Comment:
We usually have things like `"REPLACE_WITH_NAMESPACE_ID"` in here, just for consistency.